src/OpenDebugAD7/AD7Resources.Designer.cs

```diff
@@ -524,5 +524,55 @@ internal class AD7Resources {
                 return ResourceManager.GetString("Warning_Scenario_TerminateProcess", resourceCulture);
             }
         }
+
+        /// <summary>
+        ///   Looks up a localized string similar to Failed to get memory context.
+        ///.
+        /// </summary>
+        internal static string Error_GetMemoryContext {
```

Comment:
>internal [](start = 8, length = 8)

I think you must have hand edited this file and the .resx file, because you wound up putting your .resx changes in the wrong .resx file.

Reply:
Yes, I did. Thank you for catching that.